VAMobile/documentation/docs/Engineering/FrontEnd/DevelopmentBestPractices.md

```diff
@@ -0,0 +1,127 @@
+# Development Best Practices
+
+Our team follows a set of best practices around tickets, branches, and PRs to ensure a smooth development process and high quality code. Here we'll describe the key points to keep in mind.
+
+## Tickets
+
+We use tickets to track all our engineering work. They precisely specify the work to be performed, allow stakeholders to track progress, and guide QA in assessing the resulting work.
+
+### Creating tickets
+
+Every ticket must contain at least the following information:
+
+- A detailed description of the work
+- Clearly specified Acceptance Criteria
+- Screenshots or links to mockups where applicable
+- All relevant labels ("front-end" for FE work, "Health" for Health and Benefits work, "Global" for global work, etc)
+- An epic relating to the ticket's larger context
+
+If possible, choose the relevant template for your ticket. For example,  if you're creating a bug ticket, choose the Bug Report Template and include severity in the title, labels, and body.
+
+### Working on tickets
+
+All engineering work requires a ticket.
+
+Every ticket must satisfy the requirements in the "Creating tickets" section above. If information is missing, add it or contact the relevant stakeholders.
+
+Before an engineer can begin work on a ticket, it must contain an estimate. Bugs get an initial estimate of 1 to cover a short investigation. Other tickets are left to the engineers' discretion.
+
+Once everything is in place, complete the following tasks:
+
+- Assign yourself
+- Assign the appropriate QA engineer if not already assigned
+- Move the ticket to In Progress
+- Create a branch as described below in the Branches section
+
+Then you can begin work. Keep stakeholders up to date as you work by commenting on the ticket with any important details or questions. This will also preserve information so everyone can reference it in the future.
+
+On completion of your work, double check the following:
+
+- All items in Acceptance Criteria are implemented
+- Accessibility features are in place (test with iOS and Android screen readers)
+- Unit and E2E tests are updated or extended to cover your changes
```

Comment:
This would essentially mean needing to include detox changes with a PR, correct? If that's the case it's a pretty big change to our current practices. One I know Tim W. wants us to eventually make, just uncertain if we are there yet.